docs/fides/Dockerfile

```diff
@@ -1,21 +1,34 @@
-FROM python:3.9-slim-bullseye
+FROM python:3.10.6-slim-bullseye as build
 
-# Install auxiliary software
-RUN apt-get update
-RUN apt-get install -y \
-    git \
-    vim \
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
     g++ \
-    gcc
+    gnupg \
+    gcc \
+    python3-wheel \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN python3 -m venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
 
-# Update pip and install requirements
-RUN pip install -U pip
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+COPY . .
+RUN pip install -U pip && pip install . && pip install -r docs/fides/requirements.txt
```

Comment:
ok, looks like this is where fides is getting installed locally, much better solution